glossary.md

```diff
@@ -0,0 +1,133 @@
+# Glossary
+
+## Chains
+
+### Hyperchain (HC)
+A hybrid blockchain technology consisting of two blockchains: a delegated proof of stake (DPoS) child chain (CC) and a proof of work (PoW) parent chain (PC) where the child chain periodically synchronizes with the parent chain.
+
+As a PoW secured DPoS, a hyperchain benefits from the scalability advantages of a DPoS blockchain with the security advantages of a PoW blockchain.
+
+### Child Chain (CC)
+A Delegated Proof of Stake blockchain that is one half of a hyperchain. The CC is configurable by the initiator and loosely connected to the parent chain. The child chain uses the parent chain as a source of entropy and for "pinning," but the parent chain has no knowledge of the child chain.
+
+### Parent Chain (PC)
+Any PoW blockchain that fulfills the role of "the other half" of a hyperchain by providing entropy for the CC and allowing for the pinning of data from the CC. The parent chain functions independently from and has no knowledge of the child chain.
+
+## Blocks
+
+### CC Block
+Contains one keyblock and possibly one microblock.
+
+### Keyblock
+A placeholder block indicating the leader that should produce a microblock.
```

Comment:
Not sure about the placeholder comment. The keyblock contains all the state tree hashes etc so it is crucial for consensus. However, the separation into micro-block + key-block is entirely artificial and a legacy of building on top of a code base that uses BitcoinNG...